learning-PyTorch-on-IPU/pipelining/walkthrough.ipynb

```diff
@@ -2,32 +2,27 @@
  "cells": [
```

Comment:
<p>and move up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-Pytorch-fork/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>